gax/src/test/java/com/google/api/gax/batching/v2/BatcherImplTest.java

```diff
@@ -255,6 +262,67 @@ public void testWhenThresholdIsDisabled() throws Exception {
     assertThat(result.get()).isEqualTo(4);
   }
 
+  @Test
+  public void testUnclosedBatchersAreLogged() throws Exception {
+    underTest =
```

Comment:
Please add a few gc() & runFinalization() here to make sure that this test won't fail due to leaked Batcher instances from other tests

Reply:
Verifying leaked Batcher instances are garbage collected.